pkg/models/event.go

```diff
@@ -25,35 +25,29 @@ import (
  * Event struct to hold event data
  */
 type Event struct {
+	BaseObject
```

Comment:
Not sure we should be adding BaseObject back in here. We want to get rid of it, as discussed, and I think a previous PR refactored this type to remove it.

Reply:
Corrected.